src/Microsoft.ML.CodeGenerator/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -0,0 +1,111 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.ML.CodeGenerator</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.10.0" />
+    <PackageReference Include="System.CodeDom" Version="4.5.0" />
+    <PackageReference Include="NLog" Version="4.5.11" />
```

Comment:
Can you verify all these `PackageReference`s are necessary? Which can be removed?